### PR TITLE
switch image ca ref to configmap ref only (no key)

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: config/v1/types_image.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ type ImageSpec struct {
 	// AdditionalTrustedCA is a reference to a ConfigMap containing additional CAs that
 	// should be trusted during imagestream import.
 	// The namespace for this config map is openshift-config.
-	AdditionalTrustedCA ConfigMapFileReference `json:"additionalTrustedCA,omitempty"`
+	AdditionalTrustedCA ConfigMapNameReference `json:"additionalTrustedCA,omitempty"`
```
Comment (deads2k) [inline config/v1/types_image.go]: please document how the content will be used.

Comment (bparees) [reply config/v1/types_image.go]: From the existing godoc:
````
// AdditionalTrustedCA is a reference to a ConfigMap containing additional CAs that
// should be trusted during imagestream import."
```

what else are you looking for?

Comment (bparees) [reply config/v1/types_image.go]: @deads2k bump

Comment (enj) [reply config/v1/types_image.go]: Uh what key in the config map?

Comment (bparees) [reply config/v1/types_image.go]: all the keys in the configmap.

Comment (enj) [reply config/v1/types_image.go]: ah `additionalTrustedCA` made me think it was singular.

**Pull request merged**